src/omv/py/qstrdefsomv.h

```diff
@@ -551,6 +552,9 @@ Q(draw_image)
 // duplicate Q(y_scale)
 Q(alpha)
 // duplicate Q(mask)
+Q(hint)
+Q(INTERPOLATE_BILINEAR)
+Q(IMAGE_CENTER)
```

Comment:
Can you rename these with a PREFIX. Like IMAGE_HINT_BILINEAR, IMAGE_HINT_CENTER.